Adds operator token requirement to org and user docs in v2 (#5386)
influxdata/docs-v2: content/influxdb/v2/admin/users/change-password.md

## Change your password using the influx CLI

Use the [`influx user password` command](/influxdb/v2/reference/cli/influx/user/password)
to update a password for a user. Provide the following:

- An [operator token](/influxdb/v2/admin/tokens/#operator-token) using your
  [`influx` CLI connection configuration](/influxdb/v2/reference/cli/influx/#provide-required-authentication-credentials),
  `INFLUX_TOKEN` environment variable, or the `--token, -t` flag.
- The username (with the `--name, -n` flag) or the user ID (with the `--id, -i` flag).
  View usernames and IDs in the output of `influx user list`.

> - A new password with the `--password, -p` flag. If no password flag is provided

Reviewer: ```suggestion
- _Optional_: the `--password, -p` flag and the new password. If you don't provide a password flag,
```
